c/fcntl/fcntl.go

```diff
@@ -0,0 +1,42 @@
+package fcntl
```

Comment:
merge this package into `c/os`

Reply:
![image](https://github.com/user-attachments/assets/23d94838-2dcf-4b0c-b81a-338322d42cb8)

![image](https://github.com/user-attachments/assets/b937c0fb-9d34-4edb-8279-72facef8a2b6)

I have a doubt, there is already an Open function in the os, but whether the two Opens can be equivalent